Source/Core/DolphinQt2/Settings/WiiPane.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/Settings/WiiPane.h"
+
+#include <QCheckBox>
+#include <QGroupBox>
+#include <QLabel>
+#include <QSlider>
+#include <QVBoxLayout>
+#include <QWidget>
+
+#include "Core/ConfigManager.h"
+
+WiiPane::WiiPane(QWidget* parent) :QWidget(parent)
+{
+  CreateLayout();
+  ConnectLayout();
+
+  LoadConfig();
+}
+
+void WiiPane::CreateLayout()
+{
+  m_main_layout = new QVBoxLayout;
+  CreateMisc();
+  m_main_layout->setContentsMargins(0, 0, 0, 0);
+  m_main_layout->addStretch(1);
+  setLayout(m_main_layout);
+}
+
+void WiiPane::ConnectLayout()
+{
+  connect(m_wiimote_speaker_volume, &QSlider::valueChanged, this, &WiiPane::OnSaveConfig);
+  connect(m_wiimote_motor, &QCheckBox::toggled, this, &WiiPane::OnSaveConfig);
+}
+
+void WiiPane::CreateMisc()
+{
+  auto* basic_group = new QGroupBox(tr("Misc Settings"));
+  auto* basic_group_layout = new QVBoxLayout;
+  basic_group->setLayout(basic_group_layout);
+  m_main_layout->addWidget(basic_group);
+  m_wiimote_motor = new QCheckBox(tr("Enable Wiimote Motor"));
```

Comment:
This should also use "Rumble" instead of "Motor" in order to match DolphinWX (which has "Wii Remote Rumble").